core/Pos/Arbitrary/Core.hs

```diff
@@ -523,9 +534,11 @@ instance Arbitrary G.GenesisCoreData where
                 ]
         stakeDistrs <- vectorOf outerLen distributionGen
         hashmapOfHolders <- arbitrary :: Gen (Map Types.StakeholderId Word16)
+        delegation <- arbitrary
```

Comment:
Maybe delegation issuers should be related to addresses of genesis core data?

Reply:
🤷‍♂️